presto-hive/src/test/java/com/facebook/presto/hive/HiveQueryRunner.java

```diff
@@ -166,7 +166,7 @@ public static DistributedQueryRunner createQueryRunner(
                         .setNodeCount(workerCount.orElse(4))
                         .setExtraProperties(systemProperties)
                         .setCoordinatorProperties(extraCoordinatorProperties)
-                        .setBaseDataDir(baseDataDir)
+                        .setDataDirectory(baseDataDir)
```

Comment:
One more - update the argument name here too.

Reply:
good catch. Did a full grep and identified a few more files.
```
        modified:   presto-hive/src/test/java/com/facebook/presto/hive/HiveQueryRunner.java
        modified:   presto-native-execution/src/test/java/com/facebook/presto/hive/HiveExternalWorkerQueryRunner.java
        modified:   presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestHiveQueries.java
        modified:   presto-native-execution/src/test/java/com/facebook/presto/nativeworker/TestHiveTpchQueries.java
        modified:   presto-raptor/src/main/java/com/facebook/presto/raptor/filesystem/LocalFileStorageService.java
 ```